build.gradle

```diff
@@ -52,12 +52,12 @@ def integTestDependOnJniLib = false
 if (!usingRemoteCluster) {
     integTestDependOnJniLib = true
     if (!usingMultiNode) {
-        apply from: 'build-tools/knnplugin-coverage.gradle'
+       // apply from: 'build-tools/knnplugin-coverage.gradle'
```

Comment:
Why is this commented out?

Reply:
Good catch. Forgot to fix the jacoco. Its WIP